Dockerfile.dev

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:xenial
```

Comment:
why xenial?

Reply:
@AkihiroSuda saw it here: https://github.com/genuinetools/img/blob/master/.travis.yml#L2